flax/nn/normalization.py

```diff
@@ -65,6 +66,8 @@ def apply(self,
       scale_init: initializer for scale, by default, one.
       axis_name: the axis name used to combine batch statistics from multiple
         devices. See `jax.pmap` for a description of axis names (default: None).
+      replica_groups: the custom replica groups used to combine batch statistics
```

Comment:
Maybe this is known to everyone but me, but maybe we can document better what "replica groups" are?

Reply:
I think it should refer to lax.psum/pmean for more details

Reply:
I looked up the documentation and source code of `lax.psum` but I can't find the term "replica groups" there.

Apologies as maybe I am "new here" but give that JAX lacks internal docstrings, I think we need to own explaining this in the Flax API, unless we can point people to other JAX references (which would be better!)

Reply:
The `replica_groups` kwarg is added in https://github.com/google/jax/pull/2382. We might end up using a different name, though. (The name and concept are definitely not known to everyone, but the idea is something many people want to express: doing batch normalization over more than just the examples on each accelerator alone, but less than the entire global batch). TF code often uses a `distributed_group_size` keyword argument and then converts that into XLA replica groups later.

Reply:
Got it. Perhaps a link to that pull request in the docstring is a simple solution?